docker-compose.dev.yml

```diff
@@ -42,24 +42,6 @@ services:
     networks:
       - esnet
 
-  webapp:
```

Comment:
If you are going to run the app locally in another terminal window, you don't need the web app running as part of docker-compose.